### PR TITLE
`dash-network` tool

### PR DESCRIPTION
**Pull request by shumkov**
Helper which wraps docker calls. Should be installed locally.

### CODE REVIEW

File: bin/dash-network [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Terraform remote backend configuration https://www.terraform.io/docs/backends/types/s3.html
+TERRAFORM_S3_BUCKET=\"\"
+TERRAFORM_S3_KEY=\"\"\"
+TERRAFORM_DYNAMODB_TABLE=\"\"
+
+# Private and public key for SSH access to an infrastructure
+PRIVATE_KEY_PATH=\"\"
+PUBLIC_KEY_PATH=\"\"
+
+# AWS credentials are using by Terraform for creating
```
Comment (nmarley) [inline bin/dash-network]: s/using/used/

File: bin/dash-network [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+# infrastructure and by Ansible for fetching images from AWS ECR.
+#
+# You should specify credentials for the both
```
Comment (nmarley) [inline bin/dash-network]: s/for the both/for both/

File: bin/dash-network [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+# Ansible and Terraform or specify particular
+# configuration using using \`TERRAFORM_\` and \`ANSIBLE_\` prefixes
```
Comment (nmarley) [inline bin/dash-network]: s/using using/using/

File: bin/dash-network [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+# Private and public key for SSH access to an infrastructure
+PRIVATE_KEY_PATH=\"\"
+PUBLIC_KEY_PATH=\"\"
+
+# AWS credentials are using by Terraform for creating
+# infrastructure and by Ansible for fetching images from AWS ECR.
+#
+# You should specify credentials for the both
+# Ansible and Terraform or specify particular
```
Comment (nmarley) [inline bin/dash-network]: s/or specify particular/or specify a particular/

File: bin/dash-network [hunk 5]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -ea
+
+# Load configuration
+
+if [[ ! -f ".env" ]]; then
+    dotenv_example=$(<examples/.env.example)
+
+    echo "Configuration .env file not found
```
Comment (nmarley) [inline bin/dash-network]: Is this missing a `fi` somewhere?

Comment (shumkov) [reply bin/dash-network]: It is after `else` block.

Comment (nmarley) [reply bin/dash-network]: Ah, ok. 👍

**Pull request merged**